public/js/components/App.js

```diff
@@ -143,6 +151,8 @@ const App = React.createClass({
 
 module.exports = connect(
   state => ({ sources: getSources(state),
-              selectedSource: getSelectedSource(state) }),
+              selectedSource: getSelectedSource(state),
+              sidebars: getSidebarsState(state)
```

Comment:
`getSidebarDimensions` seems more specific. I say dimensions because at somepoint we'll probably be handling height as well in the narrow view.